### PR TITLE
Bump python-django to 2.2.8

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
@ehelms @wbclark this should fix your conflicts issue :)

**Comment by evgeni** [comment]
build blocked on https://github.com/theforeman/foreman-packaging/pull/4500

**Comment by ekohl** [comment]
Needs a rebase now :)

**Comment by evgeni** [comment]
it has happened.

File: packages/pulpcore/python-django/python-django.spec [hunk 1]
```diff
@@ -46,14 +46,20 @@ popd
 %install
 %py3_install
 
+# rename django-admin so we don't conflict with python2-django
+mv %{buildroot}%{_bindir}/django-admin %{buildroot}%{_bindir}/python3-django-admin
```
Comment (evgeni) [inline packages/pulpcore/python-django/python-django.spec]: the name was chosen with a fair dice (and the fact that the Fedora packages actually ship that as a "compat" symlink to `django-admin`)

**Comment by evgeni** [comment]
repoclosure is broken because nobody fixed repoclosure

**Comment by ekohl** [comment]
```
[2019-12-13T10:09:39.344Z]     package: python3-django-2.2.8-1.el7.noarch from downloaded_rpms
[2019-12-13T10:09:39.344Z]       unresolved deps:
[2019-12-13T10:09:39.344Z]          python3-sqlparse
[2019-12-13T10:09:39.344Z]          python3-pytz
```

**Pull request merged**